cmd/lsif-go/main.go

```diff
@@ -47,6 +50,7 @@ func realMain() error {
 	app.Flag("out", "The output file.").Short('o').Default("dump.lsif").StringVar(&outFile)
 	app.Flag("moduleVersion", "Specifies the version of the module defined by this project.").PlaceHolder("version").StringVar(&moduleVersion)
 	app.Flag("repositoryRoot", "Specifies the path of the current repository (inferred automatically via git).").PlaceHolder("root").StringVar(&repositoryRoot)
+	app.Flag("moduleRoot", "Specifies the module root directory relative to the repository").Default(".").StringVar(&moduleRoot)
```

Comment:
What is the difference between doing this and running the indexer in the correct root instead?

Reply:
It doesn't work: If you run it from a subdirectory of the repository, the tool uses `git` to find the real root and looks for the module there. Similarly if you set `--repositoryRoot`, as it uses that only as the starting point. 

This whole interface is kind of bad, but I didn't want to break existing use cases. Properly we shouldn't be using `git` to derive the module path, but I wanted to avoid having to update all our existing config files.

Reply:
Would now be the item to update the interface?